File: schedule-pilot-web/src/app/models/ticket-check-out/response/ticket-check-out-response.ts
import { Item } from '../../item/item';
import { TicketCheckStatus } from '../../ticket-check-in/ticket-check-status';

export class TicketCheckOutResponse {
    id: number;
    trackIdTicketOut: string;
    trackIdTicketIn: string;
    deliveryDate: string;
    returnDate: string;
    item: Item;
    ticketCheckStatus: TicketCheckStatus;
}

File: schedule-pilot-core/src/main/java/com/schedulepilot/core/response/TicketCheckOutResponse.java
package com.schedulepilot.core.response;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonInclude;
import com.fasterxml.jackson.annotation.JsonProperty;
import com.schedulepilot.core.dto.model.ItemDto;
import com.schedulepilot.core.dto.model.TicketCheckStatusDto;
import lombok.*;

import java.io.Serializable;
import java.time.LocalDateTime;

@Getter
@Setter
@Builder
@NoArgsConstructor
@AllArgsConstructor
@JsonInclude(JsonInclude.Include.NON_NULL)
@JsonIgnoreProperties(ignoreUnknown = true)
public class TicketCheckOutResponse implements Serializable {

    private Long id;
    private String trackIdTicketOut;
    private String trackIdTicketIn;
    @JsonProperty("item")
    private ItemDto itemEntity;
    @JsonProperty("ticketCheckStatus")
    private TicketCheckStatusDto ticketCheckStatusEntity;
    private LocalDateTime deliveryDate;
    private LocalDateTime returnDate;
}
